app/src/main/java/pm/gnosis/heimdall/ui/dialogs/share/RequestSignatureDialog.kt

```diff
@@ -76,6 +86,19 @@ class RequestSignatureDialog : BaseShareQrCodeDialog() {
                     activity?.scanQrCode()
                     dismiss()
                 }
+        disposables += dialog_request_signature_push.clicks()
+                .flatMapSingle {
+                    signaturePushRepository.request(safe, data())
+                            .mapToResult()
+                }
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribeForResult({
+                    context!!.toast(R.string.signature_request_sent)
+                    dismiss()
+                }, {
+                    Timber.e(it)
+                    errorSnackbar(dialog_request_signature_push, it)
```

Comment:
isn't the snackbar dimmed in the dialog? If it is maybe we can use `Context.errorToast`

Reply:
Let's wait for designs with that